tests/driver_dcf77/Makefile

```diff
@@ -0,0 +1,6 @@
+include ../Makefile.tests_common
+
+USEMODULE += dcf77
+BOARD_BLACKLIST +=  stm32f030f4-demo
```

Comment:
Why though?
If it doesn't fir the memory, that should go to `Makefile.ci`.

Reply:
region `rom' overflowed by 3712 bytes

Reply:
Yea that's what `Makefile.ci` is for:
```Makefile
BOARD_INSUFFICIENT_MEMORY := \
    stm32f030f4-demo \
    #
```